callback.js

```diff
@@ -0,0 +1,53 @@
+var http = require('http');
```

Comment:
@psinha20 You have not used the HTTP module.So why you're importing this? Please remove the unnecessary package import.